src/pgrepr/src/value/numeric.rs

```diff
@@ -54,8 +54,9 @@ impl ToSql for Numeric {
     ) -> Result<IsNull, Box<dyn Error + 'static + Send + Sync>> {
         let mut d = self.0 .0.clone();
         let scale = u16::from(numeric::get_scale(&d));
+        let is_zero = d.is_zero();
         let is_nan = d.is_nan();
-        let is_neg = d.is_negative();
+        let is_neg = d.is_negative() && !is_zero;
```

Comment:
Does this mean we don't roundtrip -0 in numeric anymore? (Did we ever? I don't know.)

Reply:
We never did; this was just making the diff in line with the PG source. This could also be an assert that negative values are never zero––maybe even should be? idk